dm/pkg/schema/tracker.go

```diff
@@ -493,6 +493,22 @@ func (tr *Tracker) CreateTableIfNotExists(table *filter.Table, ti *model.TableIn
 	return tr.dom.DDL().CreateTableWithInfo(tr.se, schemaName, ti, ddl.OnExistIgnore)
 }
 
+// SplitBatchCreateTableAndHandle will split the batch if it exceeds the kv entry size limit.
+func (tr *Tracker) SplitBatchCreateTableAndHandle(schema model.CIStr, info *[]*model.TableInfo, l int, r int) error {
+	if err := tr.dom.DDL().BatchCreateTableWithInfo(tr.se, schema, (*info)[l:r], ddl.OnExistIgnore); err != nil {
```

Comment:
what happens if len((*info)[l:r]) == 0?

prefer we create them in batch, such as 100 at a time

Reply:
I think this will only happen if txn-entry-size-limit is too small that even one create table job could not be handle. Maybe we can throw a new defined error "txn-entry-size-limit too small"? However, I think divide-and-conquer may behave faster than fixed-size batches.

Reply:
other error may happen, such as `data dir` of the embedded tidb is corrupted, might keep failing and dividing

Reply:
> However, I think divide-and-conquer may behave faster than fixed-size batches.

need bench test, if its improvement is trival compared to some fixed batch-size. perfer to use batch, it's simpler